src/desktop/apps/collect/server.tsx

```diff
@@ -10,7 +10,7 @@ export const app = express()
 
 const index = async (req, res, next) => {
   try {
-    const { APP_URL, IS_MOBILE, COLLECTION_HUBS } = res.locals.sd
+    const { APP_URL, IS_MOBILE, COLLECTION_HUB_ENTRYPOINTS } = res.locals.sd
```

Comment:
You might already be working on this, but per Abhiti's comment in slack ("we want to run two a/b tests - one for the logged out home page and one for all users on the collect page" at https://artsy.slack.com/archives/C9SATFLUU/p1569347394238200?thread_ts=1569347080.236000&cid=C9SATFLUU):

I think we need two separate experiment variables; one for the home page, one for the collection page.

Also, what do you think about adding "EXPERIMENT" to the variable name(s) to make it clear that they are experiments?

Reply:
I didn't think the two tests were going out at the same time. I'll check in with Abhiti about that. This PR was specific to the entry points on the collect page because it was my understanding that it was going out first.

Also we don't normally add experiment to the key name because if the key is in the [`running_tests.coffee`](https://github.com/artsy/force/blob/master/src/desktop/components/split_test/running_tests.coffee) file it's implied that it's an A/B test. 

If we were going to add to the variable name I would suggest "TEST" instead because "EXPERIMENT" is one of the values that the constant could produce (i.e. "control" or "experiment"). Maybe we can have a quick chat around some of the testing practices.

Reply:
Re: the two tests - I think you're right that they won't go out at the same time. I misunderstood the scope of this PR - "collection hub entrypoints" to me refers to both the entry point on the `/collect` page and the entry point on the homepage, so I expected to see one variable for each. 

If this PR is only for the collect page, then I think the name of the variable should reflect that, to avoid confusion. 

Re: adding "EXPERIMENT" - Relying on the variable being declared in another file to indicate that it is an experiment relies on the reader knowing that assumption. *I* didn't realize that assumption, and even if I did, I think I'd appreciate knowing from the variable name that it was a test, instead of having to look it up in `running_tests`. I'm totally fine with adding "TEST" instead of "EXPERIMENT".